MANIFEST.in

```diff
@@ -0,0 +1,14 @@
+include injectdll/libs/dotnet/x86/bootstrap.dll
```

Comment:
You can also remove *.dll files in this folder: https://github.com/pywinauto/injected/tree/master/backends/dotnet/bin
These files will not be needed after the injectdll wheels is available.

Reply:
Ok, It will be removed.

Reply:
I'm not sure about this point. Is this manifest not used for building wheels by `setup.py`?

Reply:
This manifest is OK, I meant that the pre-built DLLs in the [bin](https://github.com/pywinauto/injected/tree/master/backends/dotnet/bin) folders will no longer be needed after merging this PR.